pypi/build.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+mkdir -p /build
+cd /build
+
+echo "Hello from PyPi build.sh"
+
+BUILD_DIR=`pwd`
+N_CORES=`nproc`
+
+cp -r /boolector .
+
+# Setup dependencies
+cd boolector
+/bin/sh ./contrib/setup-btor2tools.sh
+/bin/sh ./contrib/setup-cadical.sh
+/bin/sh ./contrib/setup-lingeling.sh
+
+#********************************************************************
+#* boolector
+#********************************************************************
+cd ${BUILD_DIR}
+
+cd boolector
+
+./configure.sh --shared --prefix /usr
+
+cd build
+
+make -j${N_CORES}
+
+make install
+
+#********************************************************************
+#* pyboolector
+#********************************************************************
+cd ${BUILD_DIR}
+rm -rf pyboolector
+
+# Specify path to CmakeLists.txt so setup.py can extract the version
+export CMAKELISTS_TXT=/boolector/CMakeLists.txt
+
+cp -r /boolector/pypi pyboolector
+
+# Grab the main license file
+cp /boolector/COPYING pyboolector/LICENSE
+
+cd pyboolector
+
+for py in cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39; do
+  echo "Python: ${py}"
+  python=/opt/python/${py}/bin/python
+  cd ${BUILD_DIR}/pyboolector
+  rm -rf src
+  cp -r ${BUILD_DIR}/boolector/src/api/python src
+  sed -i -e 's/override//g' \
+     -e 's/noexcept/_GLIBCXX_USE_NOEXCEPT/g' \
+     -e 's/\(BoolectorException (const.*\)/\1\n    virtual ~BoolectorException() _GLIBCXX_USE_NOEXCEPT {}/' \
+       src/pyboolector_abort.cpp
```

Comment:
What's this bit for?

Reply:
In order to support a broad range of Linux distributions, pre-compiled Python libraries are built on specific older Linux distros with older compilers (Python defines a manylinux docker image based on CentOS). The supported compiler versions have limits on the C++ dialects supported (override, noexcept as a keyword, etc). This filter patches the code to comply with the compiler-supported C++.